documentation/start-documenting.rst

```diff
@@ -64,79 +64,109 @@ Building the documentation
 
 .. highlight:: bash
 
-The toolset used to build the docs is written in Python and is called Sphinx_.
-Sphinx is maintained separately and is not included in this tree.  Also needed
-are blurb_, a tool to create :file:`Misc/NEWS` on demand; and
-python-docs-theme_, the Sphinx theme for the Python documentation.
+To build the documentation, follow the steps in one of the sections below.
+You can view the documentation after building the HTML
+by opening the file :file:`Doc/build/html/index.html` in a web browser.
 
-To build the documentation, follow the instructions from one of the sections
-below.  You can view the documentation after building the HTML by pointing
-a browser at the file :file:`Doc/build/html/index.html`.
+.. note::
 
-You are expected to have installed the latest stable version of
-Sphinx_ and blurb_ on your system or in a virtualenv_ (which can be
-created using ``make venv``), so that the Makefile can find the
-``sphinx-build`` command.  You can also specify the location of
-``sphinx-build`` with the ``SPHINXBUILD`` :command:`make` variable.
+   The following instructions all assume your current working dir is
+   the ``Doc`` subdirectory in your :ref:`CPython repository clone <checkout>`.
+   Make sure to switch to it with ``cd Doc`` if necessary.
 
 
-.. _building-using-make:
+.. _doc-create-venv:
+
+Create a virtual environment
+----------------------------
 
-Using make / make.bat
----------------------
+.. _doc-create-venv-unix:
 
-**On Unix**, run the following from the root of your :ref:`repository clone
-<checkout>` to build the output as HTML::
+**On Unix platforms** that support :program:`make`
+(including Linux, macOS and BSD),
+you can create a new :mod:`venv` with the required dependencies using::
 
-   cd Doc
    make venv
-   make html
 
-or alternatively ``make -C Doc/ venv html``.  ``htmlview`` can be used
-instead of ``html`` to conveniently open the docs in a browser once the
-build completes.
+Building the docs with :program:`make` will automatically use this environment
+without you having to activate it.
+
+.. _doc-create-venv-windows:
+
+**On Windows**, or if not using :program:`make`,
+`create a new virtual environment <venv-create_>`__ manually.
+Always be sure to `activate this environment <venv-activate_>`__
+before building the documentation.
+
+
+.. _building-using-make:
+.. _using-make-make-bat:
+.. _doc-build-make:
+
+Build using Make / make.bat
```

Comment:
```suggestion
Build using make / make.bat
```

Reply:
I wasn't initially sure about this, but the [official website for GNU Make](https://www.gnu.org/software/make/) (which is used on the two primary platforms in question) always refers to the name of the software as capitalized Make (as opposed to the ``make`` command), so I followed that convention here.

That said, arguably it is referring to the (unformatted) lowercase ``make`` command rather than the program name, which would be consistent with the general usage of `make.bat`, so we may as well go ahead with this change. Not that it matters *that* much...something something the hardest problems in programming :)

Reply:
When I checked I ended up on https://www.gnu.org/software/make/manual/make.html, which *always?) uses lowercase except in Title Case, but it is also referring to the monospace font `make` command.

Well, either is fine, take your pick :)